spring-multirabbit/src/main/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitAutoConfiguration.java

```diff
@@ -236,17 +237,13 @@ private void registerContainerFactoryBean(final String name,
         /**
          * Register the RabbitAdmin bean (to enable context changing with Rabbit annotations).
          */
-        private void registerRabbitAdminAndAliases(final String name,
-                                                   final RabbitAdmin rabbitAdmin,
-                                                   final Set<String> aliases) {
-            final String beanName = name + MultiRabbitConstants.RABBIT_ADMIN_SUFFIX;
+        private void registerRabbitAdmins(final String name, final ConnectionFactory connectionFactory) {
+            final String beanName = name + RABBIT_ADMIN_SUFFIX;
+            RabbitAdmin rabbitAdmin = new RabbitAdmin(connectionFactory);
             rabbitAdmin.setApplicationContext(applicationContext);
-            rabbitAdmin.afterPropertiesSet();
             rabbitAdmin.setBeanName(beanName);
+            rabbitAdmin.afterPropertiesSet();
             beanFactory.registerSingleton(beanName, rabbitAdmin);
-            if (aliases != null && !aliases.isEmpty()) {
```

Comment:
I didn't get why the aliases were removed.

Reply:
Hi @rwanderc.
Changing the way we create the Admins, preventing them from being overwritten, aliases were no longer needed. That's why I removed them.

Reply:
Thanks @ocristian for clarifying...
But the reason for the aliases is not exactly about the issue that they were being overwritten.
It's mainly related to external integrations (e.g. internal libs) offering admins with other naming conventions or shared, which will not be covered by current logic.
I'm fine with proceeding this way anyway, but we need to keep an eye on this use case if it eventually happens to be indeed an issue with external integrations.